internal/wire/msg_body.go

```diff
@@ -40,6 +42,8 @@ type MsgBody interface {
 // indicating that connection was closed by the client.
 var ErrZeroRead = errors.New("zero bytes read")
 
+const kFlagBitSize = 4
```

Comment:
No, it's the size of the flagbits that appear immediately after the header. This has to be checked to know if a message has checksum present or not. I used a constant to avoid ambiguity

Reply:
Makes sense, thanks for explaining